c7n/resources/securityhub.py

```diff
@@ -319,6 +319,8 @@ class PostFinding(Action):
 
     permissions = ('securityhub:BatchImportFindings',)
 
+    resource_type = ""
```

Comment:
Is it alright to have this as blank? It seems that on line 554 that would result in a blank key. I understand the usage of Other on 568 but not super confident about this one.

ref for other: 
https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-findings-format.html#asff-resourcedetails-other

Reply:
this is the base class, i'm just noting here as a class attribute, all resources use subclasses and change that value, which are validated as part of test-policy meta test.

Reply:
Thanks for confirming! Makes sense to me. Was thinking there could be something like that going on though did not see the connection before. I see it now.